pkg/server/server.go

```diff
@@ -166,7 +166,7 @@ type Server struct {
 	K8sProxyConfig                      *proxy.Config
 	KnativeChannelCRDLister             ResourceLister
 	KnativeEventSourceCRDLister         ResourceLister
-	KubeAPIServerURL                    string
+	KubeAPIServerURL                    string // JS global only. Not used for proxying.
```

Comment:
I think we should have a followup story which would extract the `Server` type into the `pkg/serverconfig/types.go` and add proper go docs to each of these fields to clarify their usage, maybe even with an example. I know that we are not un/marshalling the type anywhere in the code but I feel that we came to a point where we need to have docs for our types cause its hard to keep track with all details.
WDYT ?

Reply:
Yeah, makes sense to me.